lib/extractor/types.ts

```diff
@@ -29,7 +35,7 @@ export interface OciArchiveLayer {
 
 export interface OciArchiveManifest {
   schemaVersion: string;
-  config: string;
+  config: { digest: string };
```

Comment:
Oops, do we have a coverage for this field? Actually do we need it?
Just noticing the type changed but no tests did!

Reply:
So, how we use it, is to extract `imageId`, like this: 
```
export function getImageIdFromManifest(manifest: OciArchiveManifest): string {
  try {
    return manifest.config.digest;
  } catch (err) {
    throw new Error("Failed to extract image ID from archive manifest");
  }
}
```
So if the `config` field is missing for example, it will throw. 
As for test coverage, I've added tests for image id, which covers that prop, like here: https://github.com/snyk/snyk-docker-plugin/pull/224/files#diff-745b4ac01a23bcadb4066e4c8fc69350R113
Do you think that's enough?

Reply:
Ah yes amazing, I missed it! Thanks 🙏